### ISSUE TITLE
ENHANCEMENT - Option to set number of days to start Tasks (By default)

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#239]
### Expected Behaviour:  Option to set Number of Days to start tasks by default. Now that you cn assign future tasks based on results, it should be important to have a way to set date to start tasks automatically.

![image](https://user-images.githubusercontent.com/29519901/39161844-483b1472-4738-11e8-9123-595d3aaaca0b.png)

### Steps to reproduce the problem

(Optional) If this is reproducible on a Smap hosted server then include

**Comment by nap2000** [comment]
OK so what you want is that this "number of days" value is added to the creation date of any task and becomes the start date?   Presumably this could replace the explicit "date" in #214

**Comment by atl2017** [comment]
No. What I mean is to have an alternative. Currently is fixed (week). My proposal is to have the chance to set.

**Comment by atl2017** [comment]
It could be set by Organisation in other, as a selection list:

Default Start Time for Tasks:
- Same Day
- Following Day
- Week from now
- Custom:   ____ Days from Now.

This data is overrode when #214 is implemented and remains as default for manual creation of Tasks

![image](https://user-images.githubusercontent.com/29519901/39261909-8604419c-4883-11e8-83b0-42659a6519b5.png)

**Comment by atl2017** [comment]
It was written before you include this feature in Task Group creation based on collected data. So it could be closed or maintain to consider Manual Tasks
